ynr/apps/bulk_adding/templates/bulk_add/sopns/add_form.html

```diff
@@ -39,6 +39,23 @@ <h2>Add candidates to {{ election_obj.name }}: <a href="{{ post_election.get_abs
   </ol>
   </details>
 
+  {% if post_election.rawpeople and post_election.rawpeople.source_type == "parsed_pdf"%}
+    <div class="panel callout">
+      <details open>
+        <summary>
+          🤖 This SoPN has been parsed by a bot 🤖
+        </summary>
+        <p>
+          This is a new feature that might not work perfectly.
```

Comment:
Do you think its worth adding a sentence to explain that what we want the user to do here is check/verify the bot's work against the SOPN?

Reply:
Yeah, good idea. Not really sure if what I've added is enough / the right wording. Maybe we can chat about it tomorrow.